### PR TITLE
[api-docs] Update docfx config to support new proj

### PR DESCRIPTION
**Pull request by sean-gilliam**
This PR updates the docfx configuration file to support .netStandard
project files.

### CODE REVIEW

File: docs/docfx.json [hunk 1]
```diff
@@ -23,7 +23,10 @@
           "src": "../src"
         }
       ],
-      "dest": "api"
+      "dest": "api",
+	  "properties": {
+        "TargetFramework": "netstandard1.6"
```
Comment (alexvaluyskiy) [inline docs/docfx.json]: Why `netstandard`? Not net45?

Comment (sean-gilliam) [inline docs/docfx.json]: Unless I'm missing something, we don't have different public APIs between ```net45``` and ```netstandard```. So I chose the later.

Comment (alexvaluyskiy) [inline docs/docfx.json]: We have. For example right now we dont have Akka.IO on netcore. We don't have some members related to `System.Configuration`

**Comment by sean-gilliam** [comment]
Changed to TargetFramework net45, included missing projects, and formatted the json.

**Pull request merged**